shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelper.java

```diff
@@ -121,6 +127,43 @@ public static MessageHeader.MessageDestinationComponent getMSH5MessageDestinatio
         return messageHeader.getDestinationFirstRep();
     }
 
+    // MSH-6 - Receiving Facility
+    public static Organization getMSH6Organization(Bundle bundle) {
```

Comment:
Do these all need to be static?

Reply:
They are self contained and there's no need to instate them. Is there any reason not to use static?